### ISSUE TITLE
PDM QOL suggestion updates!

### ISSUE THREAD

**Issue by CruzClementina** [InZidiuZ/op-framework-issue-tracker#2498]
**Assurance:**
I sure have!

**Summary:**
Hi! Just a few suggestions for some QOL changes that could be made to PDM. 

TLDR (cause I know its a lot to read)
- PDM app which open the PDM Catalog on the tablets. 
- Employee Lockers
- Longer Commission Timer

**Reasons:**

1. Tablet Update:
For a long time its been a discussion on how to best inform people about the PDM catalog, without the awkward "well if you go to the city Discordia and search for PDM catalog it should pop up" or the copy and paste of the link on a notepad on the showroom floor (which look trashy imo). 
With the new addition of the tablet, I think we finally have our long-awaited solution to how to have the PDM catalog inside of the city. If it could be done, we would love for a PDM app to be added to the tablet so we can provide all of our employees with them, so they can hand them to people (or we could even have a kiosk with a built-in one that just stays at PDM.) For easy customer access to the catalog.

2. Employee Lockers:
Location:
![image](https://user-images.githubusercontent.com/106713037/190939129-13649a4a-d0bc-45a6-9326-dce1e26d6782.png)
Allows us to have space to keep things that we need specifically while at PDM; lockpicks (for when the cars scuff out in the sell bay), gas cans, repair kits, medkits (for when we get run over by Danny's), etc.. Also, Tablet storage if the above gets approved!

3. The timer is very short on how long the commission is claimed to someone, I believe it's about 60 seconds which is a very short window to be able to discuss the finance and things as well as reset the commission. If that timer could be extended a little bit, we would all appreciate it. The consensus on a time frame is like 3-5 minutes.

4. Eventually: Getting Bucks Face off of the banners and the wall pictures updated to PDM cars. (:

**Comment by DaxterPancake** [comment]
Uniform requests needs to be directed to the correct github and removed from this ticket. 

Correct github for asset related things is [here](https://github.com/InZidiuZ/legacy-assets-issue-tracker/issues).

**Comment by CruzClementina** [comment]
My bad! I knew there was another place for that but couldn't remember where - Thank youu (:

**Comment by ThisIsNotDex** [comment]
In regards to the commission timer, I think it should be set a minimum of 3 minutes, because I've already had two customers purchase vehicles from me and I had become untied from the vehicle because the amount of time had passed.

**Comment by twooot** [comment]
1. PDM app: sounds like a cool idea
2. Employee Lockers: not happening
3. Timer to 5 minutes: sure

**Comment by twooot** [comment]
Extended the timer to 5 minutes. I also removed the global color shuffle every minute and holding e to set commission will now only cycle the color of that car instead of all cars

**Comment by ThisIsNotDex** [comment]
W TWOOT

**Comment by CruzClementina** [comment]
lol, thank you for getting Bucks's face off of every surface and extending the timer!
Let me know if there can be a further conversation about the app idea (:

**Comment by RiZiH** [comment]
Outfits needs to be updated

**Comment by twooot** [comment]
> Outfits needs to be updated

what do you mean by that

**Comment by DaxterPancake** [comment]
You have a hoodie, a T-shirt and now a badge that hangs around your neck. We are throwing around the idea of a pin to fit onto suits but you don't need any more clothing than that.

**Comment by twooot** [comment]
app is done

**Comment by RiZiH** [comment]
> You have a hoodie, a T-shirt and now a badge that hangs around your neck. We are throwing around the idea of a pin to fit onto suits but you don't need any more clothing than that.

A Salesman is wearing a collared T-shirt and a pant, from what angle do you he looks like a Salesman, just because of the PDM texture in the shirt we get recognized otherwise no one would recognize a man with T-shirt and pants to be a car salesman


we should have formal shirt for recruit, vest for salesman, suit for senior salesman, suit with vest for executive salesman
also id cards with everyone in the neck

**Comment by DaxterPancake** [comment]
That sounds more like an issue with your staff wearing innapropriate things instead of using the many available suits that the game has to offer. I will not be adding any more branded PDM clothing. There is plenty in the city currently.
